src/external/parser.ts

```diff
@@ -1,13 +1,16 @@
 import {File} from 'src/types/ast'
-
+import {
+  parse as flux_parse,
+  format_from_js_file as flux_format_from_js_file,
+} from '@influxdata/flux-lsp-browser'
 /*
   NOTE: This is a work around for flux being generated (from rust) for the browser and jest tests running in
         a node environment (this is only for handling tests). If a test requires a specific AST result
         then you will need to mock that out in the test.
 */
 export const parse = (script): File => {
   if (window) {
-    return require('@influxdata/flux').parse(script)
+    return flux_parse(script)
```

Comment:
this change is because you mocked the library?

Reply:
if that's the case, then... shouldn't this file be retired / roll this distinction to that part of the code?

Reply:
let me double check but I think I started trying to remove this when I started work on this PR by in Sept but I don't remember what happened.

Reply:
yeah I can't get the mocks to work without this file. Once we get the next update to the lsp we shouldn't need to mock these functions anymore and can easily clean this up. So I'll defer this refactoring until then.